src/js/main.js

```diff
@@ -215,8 +220,8 @@ function startProcess() {
     // translate to user-selected language
     i18n.localizePage();
 
-    GUI.log(i18n.getMessage('infoVersionOs', { operatingSystem: GUI.operating_system }));
-    GUI.log(i18n.getMessage('infoVersionConfigurator', { configuratorVersion: CONFIGURATOR.getDisplayVersion() }));
+    gui_log(i18n.getMessage('infoVersionOs', { operatingSystem: GUI.operating_system }));
+    gui_log(i18n.getMessage('infoVersionConfigurator', { configuratorVersion: CONFIGURATOR.getDisplayVersion() }));
```

Comment:
I like more the all way of `GUI.log` (use a class/object and call the method). This way is outdated/discouraged in modules world?

Reply:
That thing was causing most of the circular dependencies.

It's not Java or C#, there it literally no need to use classes here. Classes only make sense of data is stateful, this log does literally nothing more than reach into dom and append message. There is no need for it to be in class.

Reply:
GUI in general is used in soooo many places and needs splitting up. `log` should be it's own module/logic. But that's refactor for future. Now just fixing obvious issues.

Reply:
It's the log that appears in the GUI, in the upper part. We have too the console log. It's good to differentiate them.

Reply:
Open for other naming. We also have `logging` tab and as @McGiverGim have `console.log` so its rather overloaded term.